### ISSUE TITLE
add "invite/share" button to /write, like in /draw

### ISSUE THREAD

**Issue by robertschnuell** [medienhaus/medienhaus-spaces#171]

**Comment by aofn** [comment]
I can vaguely remember having discussions about how to handle this but I can't find the commits or chat messages or other references to it atm. We used to have an invite/share button for /write but removed it because of mypads, I think. 
IIRC there are some headaches with mypads and the folder structure it creates which is conflicting with ours in matrix. I think there were a few things like that.

If we just ignore this and add the pad to the list, we kind of suggest something it isn't (if a user clicks on the bin icon to delete the pad they were invited to, they might think they're deleting the pad for everyone, when really they are just deleting the referene to it in matrix). So we only wanted /write to display pads you actually 'own'. I think also something simple like displaying the `lock` icon for password protected pads wasn't easy or not possible. 

But I am with you here, I think users should be able to add pads from others here or invite others, but we should somehow display it properly or find a solution.

tl:dr: I think this could be a bigger endeavour unfortunately.
